src/components/common/YourCompany/ResolutionDate.vue

```diff
@@ -0,0 +1,229 @@
+<template>
+  <section id="resolution-date">
+    <v-row no-gutters>
+      <v-col cols="3" class="pr-2">
+        <label :class="{'error-text': invalidSection}">
+          <strong>Resolution Date</strong>
+        </label>
+        <v-chip
+          v-if="isCorrected"
+          x-small label
+          color="primary"
+          text-color="white"
+        >
+          <span>{{ editedLabel }}</span>
+        </v-chip>
+      </v-col>
+
+      <!-- Info tooltip-->
+      <v-col cols="9">
+        <span v-if="!onEditMode" class="info-text mr-1">{{ resolutionDate }}</span>
+        <v-tooltip
+          top
+          content-class="top-tooltip"
+          transition="fade-transition"
+          nudge-right="3"
+        >
+          <template v-slot:activator="{ on }">
+            <v-icon v-on="on" class="info-icon">mdi-information-outline</v-icon>
+          </template>
+          <span>If the resolution date is incorrect, it must be corrected through a correction filing.</span>
+        </v-tooltip>
+
+        <!-- Edit mode -->
+        <template v-if="onEditMode">
+          <div cols="12" sm="3" class="pr-4">
+            <label class="start-date-title title-label">Special Resolution Date</label>
```

Comment:
Special Resolution Date.. hmm this sounds very similar to what we're doing in this file:

https://github.com/bcgov/business-edit-ui/pull/365/files#diff-90146eec2ff469681fdca4ada6fcbc982b236bf55f65ce53e32c39e086ceb286

Reply:
Except that Special Resolution Date is Coop-only, and this ticket is a bit different (see comment at bottom of ticket).